bin/kano-uixinit

```diff
@@ -18,6 +18,15 @@
 
 . /usr/share/kano-toolset/logging.sh
 
+# If we are in Kano v3, jump to Dashboard Mode, unless instructed otherwise
+if [ "${DESKTOP_MODE}" != "1" ]; then
```

Comment:
yeah you're right @alex5imon , I am not sure what to do here at this initial stage.
Ideally we should start the dashboard as soon as possible so that the empty Xserver loads behind it silently. This version works fine on a PI2, should we proceed with a big TODO for now?